.github/workflows/ci.yml

```diff
@@ -0,0 +1,44 @@
+# ------------------------------------------------------------------------------
```

Comment:
It would be great to add a target with executes `git diff --exit-code` to ensure that someone has not forgotten to commit the autogenerated code.

Reply:
That's a great idea, I think it can be a task of it's own. Will add to todo.